tests/integration/kubernetes/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Microsoft Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set_runtime_class() {
+    sed -i -e "s|runtimeClassName: kata|runtimeClassName: kata-${KATA_HYPERVISOR}|" runtimeclass_workloads/*.yaml
```

Comment:
This sed is error prone because the paths to files are relative. Could you change it to use absolute path?

Reply:
Done